en/config/actuators.md

```diff
@@ -1,94 +1,251 @@
+- `Position X`: [X-position](#motor-position-coordinate-system), in metres.
+- `Position Y`: [Y-position](#motor-position-coordinate-system), in metres.
+- `Position Z`: [Z-position](#motor-position-coordinate-system), in metres.
+- `Direction CCW`: Checkbox to indicate motor spins counter-clockwise (uncheck for clockwise).
+- `Bidirectional`: Checkbox to indicate motor is [bidirectional](#bidirectional-motors) 
+- `Slew Rate`: ?
 
-### Conventions
+The X, Y, Z positions are in [FRD relative to the centre of gravity](#motor-position-coordinate-system).
 
-The following sections contain the conventions and explanations for configuring the geometry.
 
-#### Coordinate system
+#### Motor Geometry: VTOL Tailsitter
 
-The coordinate system is NED (in body frame), where the X axis points forward, the Y axis to the right and the Z axis down.
-Positions are relative to the center of gravity (in meters).
+The default tailsitter motor geometry is shown below.
+Motors have the same configuration fields as for the [multicopter geometry](#motor-geometry-multicopter).
 
-#### Control Surfaces and Servo Direction
+![Geometry motor: tailsitter vtol](../../assets/config/actuators/qgc_geometry_tailsitter_motors.png)
 
-Control surfaces use the following deflection direction convention:
-- horizontal (e.g. Aileron): up = positive deflection
-- vertical (e.g. Rudder): right = positive deflection
-- mixed (e.g. V-Tail): up = positive deflection
+#### Motor Geometry: VTOL Tiltrotor
+
+The default VTOL tiltrotor motor geometry is shown below.
+
+![Geometry motor: tiltrotor vtol](../../assets/config/actuators/qgc_geometry_tiltrotor_motors.png)
+
+Motors have most of the same configuration fields as for the [multicopter geometry](#motor-geometry-multicopter).
+There is an additional field to indicate which servo is used to tilt the motor:
+
+- `Tilted by`: The associated servo used for tilting the motor.
+  The properties of this servo are defined in the [Motor Tilt Servo Geometry](#motor-tilt-servo-geometry).
+
+
+#### Motor Geometry: Standard VTOL
+
+The default standard VTOL ("quadplane") motor geometry is shown below.
+
+![Geometry motor: standard vtol](../../assets/config/actuators/qgc_geometry_standard_vtol_motors.png)
+
+Motors have most of the same configuration fields as for the [multicopter geometry](#motor-geometry-multicopter).
+There is an additional field to indicate the direction in which the motor moves the vehicle (for a standard VTOL, the hover motors are usually set "upwards" and the pusher motor is set to "forwards").
+
+- `Axis`: One of `Upwards`, `Downwards`, `Forwards`, `Backwards`, `Leftwards`, `Rightwards`, `Custom`
+  - If `Custom` is selected, then the UI displays three additional fields for setting the motor orientation.
+
+#### Motor Geometry: Other Vehicles
 
-![Plane Deflections](../../assets/config/actuators/plane_servo_convention.png)
+Other vehicle types will define an appropriate motor geometry for their frame type.
```

Comment:
This is just a kind of placeholder. If we want we could add boat, etc here too. We might in future, or split out instructions for the different vehicle types.